pygments/lexers/cddl.py

```diff
@@ -108,7 +108,7 @@ class CddlLexer(RegexLexer):
     _re_uint = r"(?:0b[01]+|0x[0-9a-fA-F]+|[1-9]\d*|0(?!\d))"
     _re_int = r"-?" + _re_uint
 
-    flags = re.UNICODE | re.MULTILINE
+    flags = re.UNICODE | re.MULTILINE | re.VERBOSE
```

Comment:
I wouldn't recommend this, it will be surprising to anyone adding expressions to this lexer.
It looks like the `_re_id` doesn't really make use of `VERBOSE` much anyway, you can write it like that:
```
_re_id = (
    r'[$@A-Z_a-z]'
    r'(?:[\-\.]*[$@0-9A-Z_a-z]|[$@0-9A-Z_a-z])*'
)
```
and have the same splitting.

Reply:
Yes, it introduced a bug that took me one or two hour to sort out. :)
The regex for the float also use it; I'll rework it to remove `re.VERBOSE`.